lab/lab1/part1.js

```diff
@@ -139,7 +139,20 @@ Whole Wheat ... $4.49
 
 ===================== */
 
-// printMenuGroups(query7);
+
+function printMenuGroups (query) {
+  titles = _.keys(query)
+  function func (title) {
+    console.log (title)
+    outcome = _.map(query.title,_.template("<%= name %> ... <%- price %>"))
+    outcome.forEach(function(item) {
```

Comment:
I know everything else works so the only place I could think of that went wrong is line 148&149, but I couldn't figure it out...

Reply:
I think you're going to want to give a function to `_.map` on line 147 - and don't forget to use a `return` statement! If that step is forgotten, `outcome` will very likely remain `undefined`

Reply:
Aand it's still not working...